gbif/ingestion/ingest-gbif-beam/src/main/resources/elasticsearch/es-validator-schema.json

```diff
@@ -44,6 +19,7 @@
       "gbifId": {"type": "long"},
       "lastCrawled": {"type": "date"},
       "created": {"type": "date"},
+      "yearMonthGbifIdSort": {"type":  "long"},
```

Comment:
I think it's not needed for the validator? not sure how this index is used.

Reply:
added back